site/src/pug/includes/body.pug

```diff
@@ -30,6 +30,18 @@ include ./fork.pug
                 label order
 
         .content.flex#content
+            - let difficulty;
+            each swag in data
+                - difficulty = swag.difficulty + 'difficulty';
+                div.item
+                    div.title.flex
+                        h1=swag.name
+                        div(class=difficulty title=difficulty)
+                    p.swag=swag.swagType
+                    div.flex.img-container
+                        img(src=swag.image)
+                    p.desciption=swag.description
```

Comment:
should be `p.desciption!=swag.description`

Reply:
Thanks, fixed!